### ISSUE TITLE
Add vnstat for better stats

### ISSUE THREAD

**Issue by sistein** [flxn/tor-relay-configurator#8]
The possibility to use vnstat (or something else) in addition to arm/xyz would add some valuable stats.

**Comment by flxn** [comment]
Yes that would definitely be a nice addition but interfacing with the users web server will be challenging because everyones Nginx/Apache/Lighttpd/etc. setup is different. Of course I could install and configure vnstat but everyone would have to integrate it into their web server by themselves.
Of course I could bundle a lightweight webserver only for vnstat but that could lead to security issues.
Do you have any ideas?

**Comment by sistein** [comment]
Will think about this. But yeah running a webserver should be avoided.

**Comment by irl** [comment]
Please do not run vnstat on your relay. You are essentially monitoring user traffic in high resolution when you do this. This can cause you legal trouble, and if exposed on a webserver would be grounds for having your relay kicked out of the Tor network. If you would like to monitor bandwidth usage then please use nyx which will monitor the bandwidth safely, or use Relay Search to see historical statistics that have been safely aggregated.

**Comment by vergoh** [comment]
Could you clarify on how nyx monitors the bandwidth safely and vnStat doesn't? Both appear to show history data at a resolution of about one hour and neither most likely sees the actual traffic in any way.
